utils/pofix.py

```diff
@@ -175,6 +175,79 @@
+("I thought we left the elves behind at the gates way back when... It seems", "I thought we left the elves behind at the gates way back when... it seems"),
+("Ha! Now we can get those tools easily, and go back to our own caves.", "Ha! Now we can easily take their tools back to our own caves."),
+("True. I suspect these elves will try to besiege us here.", "True. I suspect these elves will try to besiege us."),
+("Shorbear tools. Eventually the sceptre was crafted, but ", "Shorbear tools. Eventually, the sceptre was crafted, but "),
+("not right, no matter what enchantment of runes", "not right no matter what enchantment of runes"),
+("I see... Well, we will not be fooled by your haggling", "I see... well, we will not be fooled by your haggling"),
+("I have made it past those elves, but they will chase me, and elvish", "I have made it past these elves, but they will chase me, and elvish"),
+("We will certainly help any warrior of Wesnoth who needs our help!", "We will certainly help any warrior of Wesnoth in need!"),
+("I’m a dragoon, with Haldric II’s personal bodyguard.", "I’m a dragoon with Haldric II’s personal bodyguard."),
+("I’m a cavalier, with Haldric II’s personal bodyguard.", "I’m a cavalier with Haldric II’s personal bodyguard."),
+("Thus Alanin escaped from his elvish pursuers.", "Thus, Alanin escaped from his elvish pursuers."),
+("Stop that! You leave my pets alone!", "Stop that! Leave my pets alone!"),
+("Nothing!? Even this magical piece of crap lets me down!", "Nothing!? Even this magical piece of troll dung lets me down!"),
+("symbols for these arcs... Well, it must describe the volcano", "symbols for these arcs... well, it must describe the volcano"),
+("And thus Rugnur died a glorious death, in the eyes of the dwarven sages — ", "And thus, in the eyes of the dwarven sages, Rugnur died a glorious death — "),
+("My Lord, the dwarves lead by Rugnur are dead,", "My Lord, the dwarves led by Rugnur are dead,"),
+("Why were you returning without the sceptre, then?", "Why were you returning without the Sceptre, then?"),
+("Rugnur sent me back, with the message that the sceptre was", "Rugnur sent me back, with the message that the Sceptre was"),
+("If the sceptre was not completed in your sight, what makes", "If the Sceptre was not completed in your sight, what makes"),
+("Krawg reached over his back with his beak, and brought out", "Krawg reached over his back with his beak and brought out"),
+("Those are the plans for the sceptre, or at least an earlier version.", "Those are the plans for the Sceptre, or at least an earlier version."),
+("Your advice is grating to my ears, not what I wish to hear... And yet,", "Your advice is grating to my ears, not what I wish to hear... and yet,"),
+("The sceptre would not be found until many generations after Haldric II,", "The Sceptre would not be found until many generations after Haldric II,"),
+("Increases ranged weapon accuracy 20%", "Increases ranged weapon accuracy by 20%"),
```

Comment:
This is a clear error of omitted word… it's not clear to me whether or not it belongs here…?

Reply:
I don't understand - why wouldn't it fit here?